sdk/burdock/query/sql/private/query.py

```diff
@@ -94,9 +97,8 @@ def _apply_noise(self, subquery, query, syms, types, sens, srs):
             sf = [("-" if desc else "") + colname for colname, desc in sort_fields]
 
             newrs.sort(sf)
-
-        return newrs.rows()
         
+        return (newrs.rows(), srs.bounds)
```

Comment:
For example: SELECT AVG(Age)

Step 0-2:  SELECT COUNT(Age) AS ca, SUM(Age) AS sa. # have bounds of dimension 2 (srs)
Step 3: SELECT sa / ca AS aa # output is dimension 1, no bounds (csrs)

~+~+~+~+~+~+~~+

Step 0: SELECT SUM(Age)
Step 3: SELECT 5 + SUM(Age) AS a5, 5 * SUM(Age), SUM(5 * Age) AS at5

CountA=1 (very noisy, close to 0), SumA=10;;  CountB=999 (less noise), SumB=9990 === AVGA = 10.0, AVGB == 10.0